Dockerfile

```diff
@@ -16,26 +16,39 @@
 
 FROM python:3.6-slim
 
-COPY . /opt/airflow/
-
 ARG AIRFLOW_HOME=/usr/local/airflow
 ARG AIRFLOW_DEPS="all"
 ARG PYTHON_DEPS=""
-ARG buildDeps="freetds-dev libkrb5-dev libsasl2-dev libssl-dev libffi-dev libpq-dev git"
-ARG APT_DEPS="$buildDeps libsasl2-dev freetds-bin build-essential default-libmysqlclient-dev apt-utils curl rsync netcat locales"
+ARG buildDeps="freetds-dev libkrb5-dev libssl-dev libffi-dev libpq-dev git"
+ARG APT_DEPS="libsasl2-dev freetds-bin build-essential default-libmysqlclient-dev apt-utils curl rsync netcat locales"
+
+ENV PATH="$HOME/.npm-packages/bin:$PATH"
+
+RUN if [ -n "${APT_DEPS}" ]; then apt install -y $APT_DEPS; fi \
+    && apt autoremove -yqq --purge \
+    && apt clean
+
+COPY . /opt/airflow/
 
 WORKDIR /opt/airflow
 RUN set -x \
     && apt update \
-    && if [ -n "${APT_DEPS}" ]; then apt install -y $APT_DEPS; fi \
+    && if [ -n "${buildDeps}" ]; then apt install -y $buildDeps; fi \
+    && curl -sL https://deb.nodesource.com/setup_11.x | bash - \
+    && apt install -y nodejs \
     && if [ -n "${PYTHON_DEPS}" ]; then pip install --no-cache-dir ${PYTHON_DEPS}; fi \
+    && pip install --no-cache-dir --upgrade pip \
```

Comment:
Can you pin the pip version? That way a new pip version won't break the image.

Reply:
+1, I think we could pin this (it is only pip)

Reply:
fixed to 19.0.1 now